### PR TITLE
add cart page

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: pages/cart-page/cart.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
+@import url(https://lazylibrary.netlify.app/components.css);
+@import url(../../../styles.css);
+@import url(../../pages/product-listing-page/products.css);
+@import url(../../pages/wishlist-page/wishlist.css);
+
+main{
+    flex-grow: 1;
+    margin: 1rem;
```
Comment (SuyashTiwariCoder) [inline pages/cart-page/cart.css:8]: You can make classes of margin 1rem and then use it accordingly .

File: pages/cart-page/cart.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    border-radius: 0.25rem;
+
+}
+
+.cart-products .card-wrapper {
+    max-width: 100%;
+    border: 1px solid var(--primary-color);
+    background-color: inherit;
+    box-shadow: 2px 2px 5px 2px var(--off-white);
+    cursor: pointer;
```
Comment (SuyashTiwariCoder) [inline pages/cart-page/cart.css:41]: same for cursor also

File: pages/cart-page/cart.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.cart-products .card-quantity {
+    font-size: 0.9rem;
+    padding-top: 0.25rem;
+}
+
+.cart-products .minus, .cart-products .plus {
+    font-weight: bold;
+    margin: 0 0.25rem;
```
Comment (SuyashTiwariCoder) [inline pages/cart-page/cart.css]: you can use px for value less then 1 rem

File: pages/cart-page/cart.css [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    width: 100%;
+}
+
+button {
+    cursor: pointer;
+    /* border: none; */
+}
+
+.button{
+    color: white;
```
Comment (SuyashTiwariCoder) [inline pages/cart-page/cart.css]: use variable for colors

File: pages/cart-page/cart.css [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    font-weight: bold;
+}
+
+.cart-transactions .discount-msg {
+    padding: 0.5rem 0;
+    font-size: 1rem;
+}
+
+.cart-transactions button {
+    margin-top: 0.5rem;
```
Comment (SuyashTiwariCoder) [inline pages/cart-page/cart.css]: use pixels for values less then 1 rem

**Review by SuyashTiwariCoder — CHANGES_REQUESTED** [review]
I have suggested some changes.

File: pages/cart-page/cart.css [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+.cart-products .button {
+    margin-top: 0.5rem;
+    padding: 0.25rem;
+    font-size: 1rem;
+    width: 100%;
+}
+
+button {
+    cursor: pointer;
+    /* border: none; */
```
Comment (t007rushi) [inline pages/cart-page/cart.css]: remove comments

File: pages/cart-page/cart.html [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+            </div>
+        </div>
+        <div class="rapid-links">
+            <div class="heading">Quick Links</div>
+            <div class="sub-heading"><a href="../product-listing-page/products.html">Products</a></div>
+            <div class="sub-heading"><a href="../wishlist-page/wishlist.html">Wishlist</a></div>
+            <div class="sub-heading"><a href="./cart.html">Cart</a></div>
+        </div>
+
+        <div class="contact">
```
Comment (t007rushi) [inline pages/cart-page/cart.html:129]: this generic you can use footer

File: pages/cart-page/cart.html [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+            <a href="/index.html
+            " class="brand-tag">JacketEX</a>
+            <div class="text">Stylish and comfortable Jackets for all occasions</div>
+            <div class="socials">
+                <a href="https://github.com/ganesh-kanchi"><img src="../../assets/github-brands.svg" alt="github logo"></a>
+                <a href="https://twitter.com/Ganesha_K_Dev"><img src="../../assets/twitter-brands.svg" alt="twitter logo"></a>
+                <a href="https://www.linkedin.com/in/ganesh-kanchi-b9bb7a207/"><img src="../../assets/linkedin-brands.svg" alt="linkedin logo"></a>
+            </div>
+        </div>
+        <div class="rapid-links">
```
Comment (t007rushi) [inline pages/cart-page/cart.html:122]: same as below

**Review by t007rushi — APPROVED** [review]

File: pages/cart-page/cart.html (hunk 7 above)
Comment (ganesh-kanchi) [reply pages/cart-page/cart.html:129]: I have used footer once to wrap all elements in footer.

**Review by SuyashTiwariCoder — COMMENTED** [review]
Approved

**Pull request merged**